Yona/Yona/Notifications/MeDashBoardMainViewController.swift

```diff
@@ -564,3 +565,32 @@ class MeDashBoardMainViewController: YonaTwoButtonsTableViewController {
     }
     
 }
+
+extension UIImage {
+    static func resizeImage(image:UIImage, targetSize:CGSize) -> UIImage{
+        let newSize = CGSize.calculateRelativeNewSize(currentSize: image.size, targetSize: targetSize)
+        // This is the rect that we've calculated out and this is what is actually used below
+        let rect = CGRect(x: 0, y: 0, width: newSize.width, height: newSize.height)
+        // Actually do the resizing to the rect using the ImageContext stuff
+        UIGraphicsBeginImageContextWithOptions(newSize, false, 1.0)
+        image.draw(in: rect)
+        let newImage = UIGraphicsGetImageFromCurrentImageContext()
+        UIGraphicsEndImageContext()
+        return newImage!
+    }
+}
+
+extension CGSize {
+    static func calculateRelativeNewSize(currentSize:CGSize, targetSize:CGSize) -> CGSize{
+        let widthRatio  = targetSize.width  / currentSize.width
+        let heightRatio = targetSize.height / currentSize.height
+        // Figure out what our orientation is, and use that to form the rectangle
```

Comment:
When I'm not mistaken, the Android app requests the user to crop a square. Can you please check that with @Sivateja-CMI and if I'm right, do the same for iOS?

Reply:
Yes Bert, I believe iOS app does that as well. We will test that in the device. But this generic function should not disturb any thing if the image is already cropped to square. 
And it is not only for images, it can be used for views in different orientations of the device (Portrait and landscape modes). This avoids the odd stretching of views when device is rotated.

Reply:
I've tested in devices, app requests the user to crop to square.